### ISSUE TITLE
A11y Fail: Accordion - Lose focus on expand/collapse all

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1386]
CENG_1951

https://react-magma.cengage.com/version/4.5.0/api/accordion/#controlled_accordion

### Issue:
In the Controlled Accordion example, when pressing the "Expand All" button, all of the accordions in the set expand. However, focus is no longer set on anything and the next Tab stop moves to the "Collapse All" button, while the "Expand All" button is in a disabled state. Then when pressing the "Collapse All" button, focus is again not set and the next Tab stop moves to the first accordion button.

### Suggestion:
For this control, consider using one button that changes its label from "Expand All" to "Collapse All". When the button is activated, focus should remain on it (which should be evident via a focus indicator).

**Comment by silvalaura** [comment]
This should update both of these examples:
- https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/accordion--expand-collapse-all storybook 
- https://next--upbeat-sinoussi-f675aa.netlify.app/api/accordion/#controlled_accordion example

**Comment by silvalaura** [comment]
Looks good ✅
